src/Uno.UI.Runtime.Skia.Win32/Devices/Input/Win32WindowWrapper.NonClientPointer.cs

```diff
@@ -0,0 +1,194 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.UI.Input;
+using Microsoft.UI.Windowing;
+using Uno.UI.Runtime.Skia.Win32.UI.Xaml.Window;
+using Uno.UI.UI.Input.Internal;
+using Windows.Graphics;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.UI.WindowsAndMessaging;
+
+namespace Uno.UI.Runtime.Skia.Win32;
+
+partial class Win32WindowWrapper : INativeInputNonClientPointerSource
+{
+	private readonly Dictionary<NonClientRegionKind, RectInt32[]> _regionRects = new();
+
+	private delegate IntPtr WndProcDelegate(IntPtr hWnd, uint msg, IntPtr wParam, IntPtr lParam);
+
+	// --- Win32 constants
+	private const int WM_NCHITTEST = 0x0084;
```

Comment:
Remove these in favor of src/Uno.UI.Runtime.Skia.Win32/UI/Xaml/Window/Win32NonClientHitTestKind.cs